### PR TITLE
Fixed the API links

### PR DESCRIPTION
**Pull request by jaeilepp**
ping @agramfort

### CODE REVIEW

**Comment by Eric89GXL** [comment]
If this is from building with https://github.com/mne-tools/mne-python/pull/2583 it should not be merged

**Comment by jaeilepp** [comment]
Now they should work...

**Comment by agramfort** [comment]
how did you fix it?

**Comment by Eric89GXL** [comment]
@agramfort probably this https://github.com/mne-tools/mne-python/pull/2583#issuecomment-153674552

File: dev/_sources/auto_examples/connectivity/plot_cwt_sensor_connectivity.txt [hunk 1]
```diff
@@ -105,294 +105,10 @@ Set parameters
     :align: center
 
 
-**Script output**:
```
Comment (Eric89GXL) [inline dev/_sources/auto_examples/connectivity/plot_cwt_sensor_connectivity.txt]: @jaeilepp it looks like you have logging at the `warning` level by default, as I do. This means the `info` logging bits won't be written out. @agramfort which way do we want it to be?

**Comment by jaeilepp** [comment]
Yeah, I just manually added the fix to my Sphinx.

**Comment by agramfort** [comment]
arfff ... ok. Too bad it has not yet been released.

merging then.

**Comment by Eric89GXL** [comment]
@agramfort I take it you're okay with the logging info being truncated (see my comment above), then?

**Comment by agramfort** [comment]
dev is fixed but looks like stable needs to be udpated too

https://mne-tools.github.io/dev/python_reference.html

**Comment by agramfort** [comment]
I think we should display full info when building the doc

**Comment by Eric89GXL** [comment]
So we need to rebuild, then, since this PR got rid of the full info.

@agramfort I look at `dev` and `stable` and they both have references in place.

**Comment by agramfort** [comment]
ok good. Looks like we have a plan

**Comment by Eric89GXL** [comment]
@jaeilepp can you rebuild the dev and stable docs with the `info` logging level?

**Comment by jaeilepp** [comment]
Yes. I'm on it.

**Pull request merged**